server.js

```diff
@@ -1,33 +1,114 @@
-import express from 'express'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+import express from 'express';
+import cors from 'cors';
+import listEndpoints from 'express-list-endpoints';
+import topMusicData from './data/top-music.json';
+
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(express.json())
+app.use(cors());
+app.use(express.json());
 
-// Start defining your routes here
+// Routes start here
 app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+  res.send('Add /endpoints to see all possible endpoints');
```

Comment:
You could maybe add some information about the API as well. Like "This is an API listing the top songs in Spotify" or something like that.